src/xdocs/config_naming.xml

```diff
@@ -1231,9 +1231,9 @@ class MyClass {
       <subsection name="Description">
         <p>Since Checkstyle 3.0</p>
         <p>
-          Checks that method and <code>catch</code> parameter names conform to a format specified
-          by the format property. By using <code>accessModifiers</code> property it is possible
-          to specify different formats for methods at different visibility levels.
+          Checks that method, lambda and <code>catch</code> parameter names conform to a format
+          specified by the format property. By using <code>accessModifiers</code> property it is
+          possible to specify different formats for methods at different visibility levels.
```

Comment:
@kazachka , please add examples of separate validations for method parameters and lambdas to examples of xdoc:
```
(o1, o2) -> o1.equals(o2), 
(e) -> e.toString().trim()
e -> e.toString().trim()
```
or similar.